plugin.program.chrome.launcher/resources/language/English/strings.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<strings>
+  <string id="30001">Add Website</string>
```

Comment:
Transifex is a different topic, this is more about finally being able to drop the old XML stuff in core. :)